assets/js/components/UserMenu/index.js

```diff
@@ -70,6 +72,13 @@ export default function UserMenu() {
 		} )
 	);
 
+	const isAutoCreatingCustomDimensionsForAudience = useSelect( ( select ) =>
+		select( CORE_FORMS ).getValue(
+			AUDIENCE_TILE_CUSTOM_DIMENSION_CREATE,
+			'isAutoCreatingCustomDimensionsForAudience'
+		)
+	);
```

Comment:
It would be nice to take a more general approach to disabling the user menu and dashboard sharing button that could apply to other modals too, but we can tackle that another day. Just making a note of it, no changes needed here!

Reply:
SGTM!